src/file_dialog.rs

```diff
@@ -964,6 +996,13 @@ impl FileDialog {
         }
     }
 
+    /// Returns the currently active directory entry.
+    ///
+    /// If no file has been highlighted, it returns the active directory.
+    pub const fn active_entry(&self) -> Option<&DirectoryEntry> {
```

Comment:
I don' know if there is a better name for this.
It shows either the highlighted file, or the active directory if there is no highlighted file.
So calling it `highlighted()` wouldn't fully describe what it does.

Reply:
Oh ye that's right, I didn't even think about that with `highlighted`. Then I am happy with the name `active_entry`.

For the sake of completeness, could you also create an `active_entries` method for when the dialog is in `SelectMultiple` mode?

If I'm not mistaken, it should be enough to iterate `self.directory_content` and filter out the items for which `selected` is true.
For performance reasons, we might also want to return an iterator directly from the function rather than a `Vec`.

Reply:
I added it as `active_selected_entries`.

Reply:
Thanks, looks good!